package.json

```diff
@@ -64,13 +64,13 @@
     "electron-devtools-installer": "2.0.1",
     "electron-packager": "8.1.0",
     "electron-rebuild": "1.2.1",
-    "eslint": "3.7.1",
-    "eslint-config-airbnb": "12.0.0",
-    "eslint-import-resolver-webpack": "0.6.0",
-    "eslint-plugin-flowtype": "^2.19.0",
-    "eslint-plugin-import": "2.0.0",
-    "eslint-plugin-jsx-a11y": "2.2.2",
-    "eslint-plugin-react": "6.3.0",
+    "eslint": "^3.8.1",
+    "eslint-config-airbnb": "^12.0.0",
+    "eslint-import-resolver-webpack": "^0.6.0",
+    "eslint-plugin-flowtype": "^2.25.0",
+    "eslint-plugin-import": "^1.16.0",
```

Comment:
Ah! Looks like they've only released it in eslint-config-airbnb-**base** so far. Sorry :)